code/drasil-code/lib/Language/Drasil/Code/Imperative/Comments.hs

```diff
@@ -7,28 +7,28 @@ import Language.Drasil
 import Database.Drasil (conceptChunkTable)
 import Language.Drasil.Code.Imperative.DrasilState (GenState, DrasilState(..))
 import Language.Drasil.CodeSpec (HasOldCodeSpec(..))
-import Language.Drasil.Printers (SingleLine(OneLine), sentenceDoc, unitDoc)
+import Language.Drasil.Printers (SingleLine(OneLine), sentenceDoc, unitDoc, PrintingInformation)
 
 import qualified Data.Map as Map (lookup)
 import Control.Monad.State (get)
 import Control.Lens ((^.))
 import Text.PrettyPrint.HughesPJ (Doc, (<+>), colon, empty, parens, render)
 
 -- | Gets a plain renderering of the term for a chunk.
-getTermDoc :: (CodeIdea c) => c -> GenState Doc
-getTermDoc c = do
+getTermDoc :: (CodeIdea c) => PrintingInformation -> c -> GenState Doc
```

Comment:
I'm thinking that maybe what should be done is that `GenState` should be extended to contain `PrintingInformation` rather than having to plumb it through like this.